languages.json

```
@@ -1148,6 +1148,12 @@
       "mime": ["text/x-python"],
       "extensions": ["py", "pyw"]
     },
+    "Pyret": {
+      "line_comment": ["#"],
+      "multi_line_comments": ["#|", "|#"],
```

```suggestion
      "multi_line_comments": ["#|", "|#"],
      "nested": true,
```
As I discovered with 8th, you have to tell tokei that your language supports nested multiline comments.